### ISSUE TITLE
setting the default dashboard

### ISSUE THREAD

**Issue by jprokopowich** [home-assistant/frontend#12027]
### The problem

Logger: homeassistant.components.websocket_api.http.connection
Source: components/websocket_api/connection.py:141
Integration: Home Assistant WebSocket API (documentation, issues)
First occurred: 12:57:07 PM (3 occurrences)
Last logged: 12:58:19 PM

[547964137920] Error handling message: extra keys not allowed @ data['mode']. Got 'storage' extra keys not allowed @ data['url_path']. Got 'by-location'
[548020382352] Error handling message: extra keys not allowed @ data['mode']. Got 'storage' extra keys not allowed @ data['url_path']. Got 'by-location'
[548020382352] Error handling message: extra keys not allowed @ data['mode']. Got 'storage' extra keys not allowed @ data['url_path']. Got 'by-device-type'

### What version of Home Assistant Core has the issue?

core-2022.3.3

### What was the last working version of Home Assistant Core?

core-2022.1.x

### What type of installation are you running?

Home Assistant OS

### Anything in the logs that might be useful for us?

```txt
Logger: homeassistant.components.websocket_api.http.connection
Source: components/websocket_api/connection.py:141
Integration: Home Assistant WebSocket API (documentation, issues)
First occurred: 12:57:07 PM (3 occurrences)
Last logged: 12:58:19 PM

[547964137920] Error handling message: extra keys not allowed @ data['mode']. Got 'storage' extra keys not allowed @ data['url_path']. Got 'by-location'
[548020382352] Error handling message: extra keys not allowed @ data['mode']. Got 'storage' extra keys not allowed @ data['url_path']. Got 'by-location'
[548020382352] Error handling message: extra keys not allowed @ data['mode']. Got 'storage' extra keys not allowed @ data['url_path']. Got 'by-device-type'
```

### Additional information

This occurs in web interface and the in the app (android and iphone). The default dashboard does change if you leave and come back to the dashboard area.

![Screenshot from 2022-03-10 13-03-17](https://user-images.githubusercontent.com/7526760/157736370-0fc77a64-7101-4f39-9fb7-e1215f51531c.png)

**Comment by balucanb** [comment]
Bump- same thing when I tried to create a new dashboard on a brand new installation on the web UI. Running the following- core-2022.3.4, supervisor-2022.03.3 and Home Assistant OS 7.4. I can create the dashboard but when I go back in to assign it as the default dashboard I get the same error as above  posts. I rebooted the Host, restarted the Supervisor and restarted the core, checked the config and restarted- The created a new dashboard did not set it as default and did not get any errors but OVERVIEW while still showing in dashboard does not show in the sidebar anymore- if I click it in Dashboards it asks if I want to make it the default but without me designating the one I created the default once I created it -it became the default.
